tests/samplesheet_multisample_bam.nf.test.snap

```diff
@@ -24,9 +24,6 @@
                 "BCFTOOLS_FILLTAGS_ANNO": {
                     "bcftools": 1.2
                 },
-                "BCFTOOLS_MERGE": {
```

Comment:
If this is the last time bcftools merge is used in the pipeline, lets remove it completely.